sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if not isinstance(count, int) or count < 1:
```

Comment:
`isinstance(count, int)` is over limiting, For example both `isinstance(3l, int)` and `isinstance(3.0, int)` would result in `False`. Even though there is no reason to accept them as valid arguments. I do not not think we need to type check things. Checking for the `count < 1` should be enough here.